pkg/plugins/autodiscovery/dockercompose/main.go

```diff
@@ -0,0 +1,121 @@
+package dockercompose
+
+import (
+	"strings"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/sirupsen/logrus"
+	"github.com/updatecli/updatecli/pkg/core/config"
+	discoveryConfig "github.com/updatecli/updatecli/pkg/core/pipeline/autodiscovery/config"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/pullrequest"
+	"github.com/updatecli/updatecli/pkg/core/pipeline/scm"
+	"github.com/updatecli/updatecli/pkg/plugins/utils/docker"
+)
+
+// Spec is a struct fill from Updatecli manifest data and shouldn't be modified at runtime unless
+// For Fields that requires it, we can use the struct DockerCompose
+// Spec defines the parameters which can be provided to the Helm builder.
+type Spec struct {
+	// RootDir defines the root directory used to recursively search for Helm Chart
+	RootDir string `yaml:",omitempty"`
+	// Ignore allows to specify rule to ignore autodiscovery a specific Helm based on a rule
+	Ignore MatchingRules `yaml:",omitempty"`
+	// Only allows to specify rule to only autodiscover manifest for a specific Helm based on a rule
+	Only MatchingRules `yaml:",omitempty"`
+	// Auths provides a map of registry credentials where the key is the registry URL without scheme
+	Auths map[string]docker.InlineKeyChain `yaml:",omitempty"`
+	// FileMatch allows to override default docker-compose.yaml file matching. Default ["docker-compose.yaml","docker-compose.yml","docker-compose.*.yaml","docker-compose.*.yml"]
+	FileMatch []string `yaml:",omitempty"`
+}
+
+// DockerCompose hold all information needed to generate helmfile manifest.
```

Comment:
`helmfile` ? Is it a copy and paste issue?

Reply:
Yes it's :/.
I reused the helmfile plugin as a template